index.html

```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
         </ul>
         </nav>
         <a class="bg_img"></a>
-  </footer>
-  </section>
+      </article>
+    </section>
+
+    <section class="works">
+      <section class="title">My Recent Works</section>
```

Comment:
- [ ] The works section header should use `hx` i.e `h2`, `h3` etc. Kindly replace this section tag with any of the hx. This sis essentially going to make your HTML structure semantically correct and improve the page SEO.